diesel/Cargo.toml

```diff
@@ -25,6 +25,11 @@ time = { version = "0.1", optional = true }
 url = { version = "1.4.0", optional = true }
 uuid = { version = ">=0.2.0, <0.6.0", optional = true, features = ["use_std"] }
 ipnetwork = { version = "0.12.2", optional = true }
+num = { version = "0.1.37", optional = true }
+num-bigint = { version = "0.1.37", optional = true }
+num-traits = { version = "0.1.35", optional = true }
+num-integer = { version = "0.1.32", optional = true }
```

Comment:
From what I can gather, the `num` crate includes `num-{bigint, traits, integer}` by default. What's your reason for (a) nor just depending on `num`, or (b) depending on `num` at all? You don't even use `extern crate num;` below.

FWIW, it seems that `bigdecimal` also depends on `num` and `num-traits`, and we should try to use the same versions.